deployment/terraform/create.go

```diff
@@ -109,7 +109,12 @@ func (t *Terraform) Create(initData bool) error {
 	// Validate the license only if we deploy app nodes;
 	// otherwise we don't need a license at all
 	if t.config.AppInstanceCount > 0 {
-		if err := validateLicense(t.config.MattermostLicenseFile); err != nil {
+		licenseData, err := os.ReadFile(t.config.MattermostLicenseFile)
+		if err != nil {
+			return fmt.Errorf("failed to read license file: %w", err)
+		}
+
+		if err := validateLicense(licenseData); err != nil {
```

Comment:
Reading from the file still seems like the right way for me. We can tweak the tests to move the license contents to a test asset file and read from it.

Reply:
I don't really see any more value in one or the other alternative, to be honest. I usually use "easier for testing" as a proxy for slightly better code, but it doesn't really matter where to get the license data from, a string or an asset file.

So I'll revert and use an asset file, then :)